src/cephadm/tox.ini

```diff
@@ -49,7 +49,7 @@ deps =
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm.py cephadmlib}
-    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 11'
+    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 7'
```

Comment:
Shouldn't we remove all `docker.io` references from `data_utils.py` (which btw are in comments) and make this check equal `0` so we ensure no `docker.io` references appear again?

And then we should add a similar `quay\.io` check, so that we continue checking whatever this check is doing?

Reply:
I'd say that makes sense